### PR TITLE
Add support for overwrite attributes

### PR DESCRIPTION
**Pull request by BrandonY**
Adds support for displaying GCS overwrite-related attributes.

Also fixes a formatting bug for output.

### CODE REVIEW

File: storage/cloud-client/notification_polling.py [hunk 1]
```diff
@@ -64,6 +64,8 @@ def summarize(message):
     bucket_id = attributes['bucketId']
     object_id = attributes['objectId']
     generation = attributes['objectGeneration']
+    overwroteGeneration = attributes['overwroteGeneration']
```
Comment (jonparrott) [inline storage/cloud-client/notification_polling.py]: our style guide is `snake_case` for variable names, even if you're plucking them from the API repr.

Comment (BrandonY) [reply storage/cloud-client/notification_polling.py]: Oh dangit, I do that every time. I just can't stop Javaing.

Comment (jonparrott) [reply storage/cloud-client/notification_polling.py]: Hahaha. :)

**Comment by jonparrott** [comment]
@BrandonY arg, this seems to fail with the system tests https://travis-ci.org/GoogleCloudPlatform/python-docs-samples/builds/280571893

Can you see about fixing it?

**Pull request merged**